src/ImGui.NET/ImGuiNative.cs

```diff
@@ -239,6 +225,23 @@ public static unsafe class ImGuiNative
         [DllImport(cimguiLib)]
         public static extern float igGetItemsLineHeightWithSpacing();
 
+        // Columns
```

Comment:
Any particular reason you moved these down lower? It makes it a little harder to review since these all seem to be identical to before.